core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -28,6 +28,7 @@
 import org.apache.iceberg.BaseMetadataTable;
 import org.apache.iceberg.BaseTable;
 import org.apache.iceberg.BaseTransaction;
+import org.apache.iceberg.HasTableOperations;
```

Comment:
Why has this class changed? Is it still needed?

Reply:
Nope. Sorry. I had asked on here but I think it got lost. I just reverted it though.

I can open those changes in another PR. They're not strictly needed for these tests (all tests still pass - but we should consider making those changes later as they are more correct that way).

Reply:
Why are those changes more correct?

Reply:
They've been reverted so once the tests pass, this is the minimum change set for these tests.